Linters/README.md

```diff
@@ -0,0 +1,15 @@
+# Run all available linters
+
+To run all available linters, execute the following command in this directory:
+
+```
+.\Invoke-AllLinters.ps1 -RootDir .. -ConfigDir $pwd
```

Comment:
why `$pwd` instead of `.`?

Reply:
PSScriptAnalyzer quirk. I can't get it to work with config from rel path....